bitnami/influxdb/templates/ingress.yaml

```diff
@@ -21,12 +21,12 @@ spec:
           {{- if eq $architecture "high-availability" }}
           - path: /write
             backend:
-              serviceName: {{ include "influxdb.fullname" $ }}-relay
+              serviceName: {{ include "common.names.fullname" $ }}-relay
```

Comment:
Missing the new features included by @javsalgar recently:
```yaml
          - path: {{ .Values.ingress.path }}
            {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
            pathType: {{ .Values.ingress.pathType }}
            {{- end }}
            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "http" "context" $)  | nindent 14 }}
```

Reply:
done!